lang/cppreference/Portfile

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=portfile:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+name                    cppreference
+version                 20190607
+maintainers             {eborisch @eborisch} \
+                        openmaintainer
+categories              lang
+license                 GFDL
+platforms               darwin
+
+homepage                https://cppreference.com/
+
+description             C++ reference pages (and Doxygen tags files)
+long_description        ${description}. Offline version.
+
+master_sites            http://upload.cppreference.com/mwiki/images/1/16/
+distfiles               html_book_${version}.tar.xz
+use_xz                  yes
+
+checksums \
+    rmd160  5b197a0107d53324d2eead5201a9cb6d51f5d581 \
+    sha256  8f97b2baa749c748a2e022d785f1a2e95aa851a3075987dfcf38baf65e0e486d \
+    size    4172152
+
+use_configure           no
+build {}
```

Comment:
If nothing is compiled then you need to add the following just below the `platforms` line:

`supported_arches noarch
`